relay-dynamic-config/src/feature.rs

```diff
@@ -1,60 +1,79 @@
-use std::borrow::Cow;
+use std::collections::BTreeSet;
 
 use serde::{Deserialize, Serialize};
 
 /// Features exposed by project config.
-#[derive(Clone, Debug, Eq, PartialEq, PartialOrd, Ord, Hash)]
+#[derive(Clone, Debug, Eq, PartialEq, PartialOrd, Ord, Hash, Serialize, Deserialize)]
 pub enum Feature {
     /// Enables ingestion of Session Replays (Replay Recordings and Replay Events).
+    #[serde(rename = "organizations:session-replay")]
     SessionReplay,
     /// Enables data scrubbing of replay recording payloads.
+    #[serde(rename = "organizations:session-replay-recording-scrubbing")]
     SessionReplayRecordingScrubbing,
     /// Enables device.class synthesis
     ///
     /// Enables device.class tag synthesis on mobile events.
+    #[serde(rename = "organizations:device-class-synthesis")]
     DeviceClassSynthesis,
     /// Enables metric extraction from spans.
+    #[serde(rename = "projects:span-metrics-extraction")]
     SpanMetricsExtraction,
     /// Apply transaction normalization rules to transactions from legacy SDKs.
+    #[serde(rename = "organizations:transaction-name-normalize-legacy")]
     NormalizeLegacyTransactions,
+
+    /// Deprecated, still forwarded for older downstream Relays.
+    #[serde(rename = "organizations:transaction-name-mark-scrubbed-as-sanitized")]
+    Deprecated1,
+    /// Deprecated, still forwarded for older downstream Relays.
+    #[serde(rename = "organizations:transaction-name-normalize")]
+    Deprecated2,
+    /// Deprecated, still forwarded for older downstream Relays.
+    #[serde(rename = "organizations:profiling")]
+    Deprecated3,
```

Comment:
Now that deprecated features are not forwarded as `Unknown(String)`, we have to list them explicitly (taken from [sentry](https://github.com/getsentry/sentry/blob/7ef1718552effcdf2de5f664882fcb395841ef9f/src/sentry/relay/config/__init__.py#L50-L60)). Listing them explicitly will also encourage us to clean up every once in a while.